src/test/java/com/scmspain/karyon/accesslog/AccessLogIntegrationTest.java

```diff
@@ -27,9 +31,10 @@
 public class AccessLogIntegrationTest {
   private static KaryonServer server;
   private static AccessLogFormatter mockedFormatter;
+  private static final  String IP_FOR_TEST = "8.8.8.8";
 
-  @BeforeClass
-  public static void setUpBefore() throws Exception {
+  @Before
+  public void setUp() throws Exception {
```

Comment:
For each test the mockFormatter is different, and mockFormatter is injected in :
  BootstrapModule[] bootstrapModules = {
      (binder) -> binder.bind(AccessLogFormatter.class).toInstance(mockedFormatter)
    };

It seems that once you have injected you cannot modify the match function more than once.